libsync/SyncMsgEngine.h

```diff
@@ -125,6 +139,9 @@ class SyncMsgEngine : public std::enable_shared_from_this<SyncMsgEngine>
     std::shared_ptr<dev::ThreadPool> m_txsWorker;
     std::shared_ptr<dev::ThreadPool> m_txsSender;
     std::shared_ptr<dev::ThreadPool> m_txsReceiver;
+    std::shared_ptr<dev::ThreadPool> m_timeAlignWorker;

```

Comment:
so many workers, maybe we can simplify these workers in the future
add a TODO tag?

Reply:
ok.

Reply:
TODO already added.